manual_models.py

```diff
@@ -50,7 +50,7 @@ def __init__(self):
     def forward(self, x):
```

Comment:
is the model compatible with input data?

Reply:
Yes, I could make it trained with this model.